src/include/basil.h

```diff
@@ -649,6 +735,13 @@ typedef struct basil_node_memory {
 	struct basil_node_memory *next;
 } basil_node_memory_t;
 
+/* This computeunit struct is added by Altair */
```

Comment:
Hi @lisa-altair , 
   What you suggested is doable, but i don't see any value in  adding another header file. I mean, since this is in OSS now, and as pointed out by @mike0042 that all contribution is equal. I can not see what benefit we'll get out of  differentiating what was given by Cray and what we or anyone added.? Anyways I am fine with whatever you guys decide :) .